vendor/bat-native-ledger/src/bat/ledger/internal/wallet/create.cc

```diff
@@ -29,7 +29,26 @@ Create::Create(bat_ledger::LedgerImpl* ledger) :
 Create::~Create() {
 }
 
-void Create::Start(ledger::CreateWalletCallback callback) {
+void Create::Start(const std::string& safetynet_token,
+      ledger::CreateWalletCallback callback) {
+  if (!safetynet_token.empty()) {
+    std::string safetynet_prefix = PREFIX_V5;
+#if defined (OS_ANDROID) && defined(ARCH_CPU_X86_FAMILY) && defined(OFFICIAL_BUILD)
+    safetynet_prefix = PREFIX_V3;
```

Comment:
Should be decouple into platform specific classes

Reply:
Issue created for that https://github.com/brave/brave-browser/issues/6200